packages/sirius-web/backend/sirius-web-application/pom.xml

```diff
@@ -219,6 +219,11 @@
 			<artifactId>org.eclipse.emf.ecore.change</artifactId>
 			<version>2.17.0</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+			<version>2.5.0</version>
+		</dependency>
```

Comment:
This dependency should be on `sirius-web-infrastructure` and `sirius-web-application` should only depend on the jar containing the Swagger annotations `io.swagger.core.v3 : swagger-annotations-jakarta : 2.2.21`